Core1Platform/AdministratingCore1/06_ExpertKnowledge.md

```diff
@@ -0,0 +1,106 @@
+# Expert knowledge
+
+This section contains procedures that you only should use, if you really know the consequences of your actions well.
+
+
+
+## Switch on &frasl; off single feature flags
+
+The Entwicklermodus (Development mode) of the engine room allows to switch on &frasl; off specific system behavior fundamentally.   
```

Comment:
```suggestion
The *Development mode* of the engine room allows to switch on &frasl; off specific system behavior fundamentally.   
```
*Development mode" option/checkbox ? Es ist ein UI-Element.
Folgender Satz (For example, ...) würde ich in der Zeile oben (also, im gleichen Abschnitt schreiben).